README.md

```diff
@@ -12,34 +13,59 @@
 - [3. Routing](#3-routing-3h)
 - [4. Services using HttpClient](#4-services-using-httpclient-3h)
 - [5. Forms and Validation](#5-forms-and-validation-3h)
-- [6. Authorization and Guards](#6-authorization-and-guards-3h)
-- [7. Simple NGRX](#7-simple-ngrx-6h)
+- [6. Authorization and Guards](#6-authentication-authorization-and-guards-3h)
+- [7. Simple NgRx](#7-simple-ngrx-6h)
 - [8. Material Components](#8-use-material-components-1h---2h)
 - [OPT-1. Internationalization](#opt-1-internationalization)
-- [OPT-2. Normalized NGRX](#opt-2-normalized-ngrx)
+- [OPT-2. Normalized NgRx](#opt-2-normalized-ngrx)
 - [OPT-3. Unit Testing](#opt-3-unit-testing)
 - [OPT-4. E2E Testing](#opt-4-e2e-testing)
 - [OPT-5. Component Library Development](#opt-5-component-library-development)
+- [OPT-6. Mono-Repository with Nx](#opt-6-mono-repository-with-nx)
+- [Frontend Development - Learn More](#frontend-development---learn-more)
 
-## Working Mode
+## Working Mode (Must-Read)
 
 The road-map consists of several steps. In each step, a set of theoretical concepts are explored, supported by reference documentation, book chapters, tutorials and videos. In parallel, a simple application will be built with the learned concepts: the *Online Shop* application.
 
 After the learning material for a given step was sufficiently explored, either some new functionality will be added to this application or old functionality will be refactored.
 
-All the code written must be published on GitHub. Access the [this link](https://classroom.github.com/a/Ch8FVRTf) to create your own repository. Commits must be pushed when each individual chapter is finished. In order to request a code review from the trainers, you must [open a pull request](https://help.github.com/en/articles/creating-a-pull-request) from the `develop` to the `master` branch.
+All the code written must be published on GitHub. Access [this link](https://classroom.github.com/a/xWWQ0td5) to create your own repository. 
+Commits must be pushed when each individual chapter is finished.
+
+**In order to request a code review from the trainers**, you must [open a pull request](https://help.github.com/en/articles/creating-a-pull-request) from the `develop` to the `master` branch. Inform them in your **daily standup** of this or through a PM.
+**Once the Pull Request is approved** by the trainers, merge it into master and create another branch from master to continue work.
+*Repeat ad infinitum* (until training end's)
 
 ## Environment
 
 You can work using your local environment:
 - You need to install [NodeJS](https://nodejs.org/en/) and [VSCode](https://code.visualstudio.com/download).
+- You can use [Webstorm 30 Day Trial](https://www.jetbrains.com/webstorm/) as an alternative to VSCode
```

Comment:
Don't mention the trial. Some people have university licenses or licenses paid by msg.

Reply:
And, they can use them if they have them?

Reply:
Done